tslint.json

```diff
@@ -0,0 +1,12 @@
+{
+  "extends": "tslint:recommended",
+  "rules": {
+    "indent": [
+      true,
+      "tabs"
+    ],
+    "variable-name": [
+      false
```

Comment:
I don't want to change the private API, so I allowed undescore (`_`) prefixes for private variables... But I can also change this...